### PR TITLE
Added the logo to the README 

### PR DESCRIPTION
**Pull request by palewire**
screenshot from 2018-03-23 15-45-27

### CODE REVIEW

**Comment by jakevdp** [comment]
Looks like this has conflicts with recent changes. Can you rebase on master?

**Comment by palewire** [comment]
Done.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+[Vega-Lite](https://github.com/vega/vega-lite) JSON specification. This elegant
+simplicity produces beautiful and effective visualizations with a minimal amount of code. *Altair is developed by [Jake Vanderplas](https://github.com/jakevdp) and [Brian
 Granger](https://github.com/ellisonbg) in close collaboration with the [UW
 Interactive Data Lab](http://idl.cs.washington.edu/).*
 
-With Altair, you can spend more time understanding your data and its meaning. Altair's
-API is simple, friendly and consistent and built on top of the powerful
-[Vega-Lite](https://github.com/vega/vega-lite) JSON specification. This elegant
-simplicity produces beautiful and effective visualizations with a minimal amount of code.
+[![build status](http://img.shields.io/travis/altair-viz/altair/master.svg?style=flat)](https://travis-ci.org/altair-viz/altair)
```
Comment (jakevdp) [inline README.md]: Please move the badge to right below the heading in the README

**Comment by palewire** [comment]
I think I got what you're aiming for. 

screenshot from 2018-03-24 14-43-42

**Comment by jakevdp** [comment]
Nice! Thanks :smile:

**Pull request merged**